pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>com.expediagroup</groupId>
     <artifactId>eg-oss-parent</artifactId>
-    <version>1.3.1</version>
+    <version>2.3.0</version>
```

Comment:
Add this version update to the CHANGELOG. We also need a line explaining that Docker images are now built using Jib rather than the other plugin. Although it shouldn't have any end-user facing impact I don't think it hurts to increment the version number of Beekeeper to `3.1.0` to indicate this change which is more than a bug fix.